docs/test_plan.rst

```diff
@@ -0,0 +1,67 @@
+################
+CCXCon Test Plan
+################
+
+CCXCon is not a user-facing application, so this document will focus on its
+API and its integration with Teacher's Portal and edX.
+
+These are the API endpoints:
+
+Courses Collection [/api/v1/coursexs/]
+Course [/api/v1/coursexs/{course_uuid}/]
+Modules collection [/api/v1/coursexs/{course_uuid}/modules/]
+Module [/api/v1/coursexs/{course_uuid}/modules/{module_uuid}/]
+CCX Collection [/api/v1/coursexs/{course_uuid}/ccxs]
+CCX [/api/v1/coursexs/{course_uuid}/ccxs/{ccx_uuid}]
+Modules collection [/api/v1/coursexs/{ccx_uuid}/modules]
+Module membership [/api/v1/ccxs/{ccx_uuid}/modules/{module_uuid}]
```

Comment:
Is it possible to clean up the formatting here. Is this is a list, or maybe a table?